Gemfile.lock

```diff
@@ -88,9 +74,8 @@ DEPENDENCIES
   marc
   rake
   rsolr
-  solr_wrapper!
   traject (= 3.0.0.alpha.2)
```

Comment:
traject 3.0 did not work?

Reply:
with 1f463ce18 this is solved

Reply:
hmm thought i did a bundle install... maybe i need to specifically do bundle update?

Reply:
so that is outdated, i think i caught this mistake after-the-fact